### PR TITLE
Now logouts out of site,  not all of Google

### PR DESCRIPTION
**Pull request by apluscs**
Not tested on deployment but should when this is merged

### CODE REVIEW

File: src/main/webapp/about.js [hunk 1]
```diff
@@ -27,17 +27,17 @@ function checkLogin() {
       for(let element of restrictedTabs) {
         element.classList.remove('disabled');
       }
-      document.getElementById('login-tab').innerText = 'Logout';
+      document.getElementById('logout-tab').removeAttribute('hidden');
       // If not registered, redirect to registration.
       let loginInfo = getLoginInfo();
       loginInfo.then(getUserOrRedirectRegistration);
+      document.getElementById('login-tab').href = status.loginUrl;
```
Comment (meagancheese) [inline src/main/webapp/about.js]: Shouldn't this be logout-tab?

File: src/main/webapp/about.js [hunk 2]
```diff
@@ -27,17 +27,17 @@ function checkLogin() {
...
     } else {
       restrictedTabs = document.getElementsByClassName('restricted-tab');
       for(let element of restrictedTabs) {
         element.classList.add('disabled');
       }
-      document.getElementById('login-tab').innerText = 'Login';
+      document.getElementById('login-tab').removeAttribute('hidden');
```
Comment (meagancheese) [inline src/main/webapp/about.js:39]: You still need to set the link on the login-tab.

File: src/main/webapp/authentication.js [hunk 3]
```diff
@@ -13,16 +13,25 @@
...
-  return fetch('/login').then(response => response.json());
+  return fetch('/login').then(response => {
+    return response.json();
+  });
+}
+
+function logout(){
+  fetch('/logout').then(response => {
+    if(response.redirected) {
+      console.log('redirected ' + response.url);
```
Comment (meagancheese) [inline src/main/webapp/authentication.js]: Remove

File: src/main/webapp/script.js [hunk 4]
```diff
@@ -26,19 +26,19 @@ function checkLogin() {
       for(let element of restrictedTabs) {
         element.classList.remove('disabled');
       }
-      document.getElementById('login-tab').innerText = 'Logout';
+      document.getElementById('logout-tab').removeAttribute('hidden');
       // If not registered, redirect to registration.
       let loginInfo = getLoginInfo();
       loginInfo.then(getUserOrRedirectRegistration);
+      document.getElementById('login-tab').href = status.loginUrl;
```
Comment (meagancheese) [inline src/main/webapp/script.js]: This should be logout-tab

File: src/main/webapp/script.js [hunk 5]
```diff
@@ -26,19 +26,19 @@ function checkLogin() {
...
     } else {
       document.getElementById('login-message-container').style.display = 'inline';
-      document.getElementById('login-message').innerHTML = 'To get started, please <a href="' + status.changeLogInStatusURL + '">login</a>.';
+      document.getElementById('login-message').innerHTML = 'To get started, please <a href="' + status.loginUrl + '">login</a>.';
       restrictedTabs = document.getElementsByClassName('restricted-tab');
       for(let element of restrictedTabs) {
         element.classList.add('disabled');
       }
-      document.getElementById('login-tab').innerText = 'Login';
+      document.getElementById('login-tab').removeAttribute('hidden');
```
Comment (meagancheese) [inline src/main/webapp/script.js:41]: Set this link

**Review by meagancheese — CHANGES_REQUESTED** [review]
Some changes needed for what's happening when user is logged in vs. logged out.

**Review by meagancheese — APPROVED** [review]

**Review by GrantComm — APPROVED** [review]

**Review by acmcarther — APPROVED** [review]

**Pull request merged**